src/main/java/io/github/fablabsmc/fablabs/api/fluidvolume/v1/volume/api/FixedSizedFluidVolume.java

```diff
@@ -0,0 +1,52 @@
+package io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.api;
+
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.math.Fraction;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.nbt.CompoundTag;
+
+/**
+ * a fluid contained with a fixed size.
+ */
+public class FixedSizedFluidVolume extends FluidVolume implements FixedSizeFluidVolume {
```

Comment:
```suggestion
public class SimpleFixedSizeFluidVolume extends FluidVolume implements FixedSizeFluidVolume {
```

So you can distinguish between the interface and the impl

Reply:
good name